aws-lc-rs/src/ec/key_pair.rs

```diff
@@ -318,5 +324,6 @@ mod tests {
         assert_eq!(key_pair.evp_pkey, key_pair_5208.evp_pkey);
         assert_eq!(key_pair.evp_pkey, key_pair_5915.evp_pkey);
         assert_eq!(key_pair_5208.evp_pkey, key_pair_5915.evp_pkey);
+        assert_eq!(key_pair_5915.algorithm, &ECDSA_P256_SHA256_FIXED_SIGNING);
```

Comment:
This doesn't actually invoke the function that you added. I'll push a commit to update an "external" test to invoke this function.